src/wlapplication.cc

```diff
@@ -782,6 +790,8 @@ void WLApplication::run() {
 	GameLogicThread game_logic_thread(&should_die_);
 
 	if (game_type_ == GameType::kEditor) {
+		check_crash_reports();
```

Comment:
Could this be refactored so that it could be moved before the `if()`, instead of calling it in each case separately?

Reply:
The problem with that is that if `check_crash_reports()` creates a main menu then we want to keep it around only in the `else` block (instead of creating a new one) but not in any of the `if` blocks. So if we only call `check_crash_reports` once then all blocks except the last still have to call `menu.reset()` to delete the undesired main menu before starting their own activity, which wouldn't gain anything IMHO.

Reply:
Thank you. Sorry for the nitpicking. :)